coralogix-eventbridge-policy/template.yaml

```diff
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: The module will create a policy on a given EventBridge Event Bus so that Coralogix can send data to it.
+Parameters:
+  EventBusArn:
+    Type: String
+    Description: The ARN corresponding to the Event Bus that will receive events via the PutEvents method.
+    AllowedPattern: '^arn:aws[\w-]*:events:[a-z]{2}-[a-z]+-[\w-]+:[0-9]{12}:event-bus\/[\.\-_A-Za-z0-9]+$'
+    MaxLength: 2048
+  CoralogixRegion:
+    Type: String
+    Description: The Coralogix region, possible options are [Europe, Europe2, India, Singapore, US, US2, Custom]
+    AllowedValues:
```

Comment:
Same here need to change the regions according to the coralogix region here [coralogix domains](https://coralogix.com/docs/coralogix-domain/)